### PR TITLE
Enable and fix some lint errors in circlci

### PR DESCRIPTION
**Pull request by rjoleary**

### CODE REVIEW

File: pkg/uroot/test/foo/foo.go [hunk 1]
```diff
@@ -25,12 +25,12 @@ var (
 )
 
 var (
-	groupedDeclOnlyIntf interface{}
-	nonConstantAssign   = fmt.Errorf("foo")
```
Comment (hugelgupf) [inline pkg/uroot/test/foo/foo.go]: Please undo this file's changes

File: cmds/switchroot/switch_root.go [hunk 2]
```diff
@@ -174,14 +174,14 @@ func specialFS(newRoot string) error {
...
+	log.Printf("switchroot: moving mounts")
 	if err := specialFS(newRoot); err != nil {
-		return fmt.Errorf("switch_root: moving mounts failed %v", err)
+		return fmt.Errorf("switchroot: moving mounts failed %v", err)
 	}
 
-	log.Printf("switch_root: Changing directory")
+	log.Printf("switchroot: Changing directory")
 	if err := unix.Chdir(newRoot); err != nil {
-		return fmt.Errorf("switch_root: failed change directory to new_root %v", err)
```
Comment (hugelgupf) [inline cmds/switchroot/switch_root.go]: Please undo switch_root rename

File: pkg/uroot/test/foo/foo.go (hunk 1 above)
Comment (rjoleary) [reply pkg/uroot/test/foo/foo.go]: Done.

File: cmds/switchroot/switch_root.go (hunk 2 above)
Comment (rjoleary) [reply cmds/switchroot/switch_root.go]: Done.

**Comment by rjoleary** [comment]
PTAL

**Pull request merged**